res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -331,6 +351,20 @@ function sendShortMsg(control,midino,value) {
     midi.sendShortMsg(control, midino, value);
 }
 
+function parameterSoftTakeOver (group, control, value) {
```

Comment:
The soft take over in Mixxx only works with engine.setvalue, not with engine.setparameter. There is a documented request in the launchpad